src/components/icons/MozillaLogo.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
+export let size = {width: 100, height: 36};
```

Comment:
I'd remove `height` from this and only allow the user to specify width, if possible, to prevent warp